requirements/edx/paver.txt

```diff
@@ -5,3 +5,4 @@ lazy==1.1
 path.py==7.2
 watchdog==0.7.1
 python-memcached
+libsass==0.10.0
```

Comment:
I didn't need to install any system packages -- the pip install downloads and compiles the C library it depends on.